en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -52,7 +54,9 @@ Let's create a new REST API component as follows:
 
 5. Click **Next**.
 
-In the **REST API** dialog box that opens, you can specify whether you want to save the REST API configuration in a Choreo-managed repository or a repository managed by you. In this tutorial, let's select **Choreo-managed repository** and click **Create**.
+    In the **REST API** dialog box that opens, you can specify whether you want to save the REST API configuration in a Choreo-managed repository or a repository managed by you. **Choreo-managed repository** is selected by default. For this tutorial, let's leave this selection unchanged.
```

Comment:
```suggestion
    In the **REST API** dialog box that opens, you can specify whether you want to save the REST API configuration in a Choreo-managed repository or a repository managed by you. The **Choreo-managed repository** is selected by default. For this tutorial, let's leave this selection unchanged.
```

Reply:
If we use "The" at the start, won't we have to have the word "option" after **Choreo-managed repository**?

Reply:
Changed to "The **Choreo-managed repository** option is selected by default."